### DISCUSSION TITLE
Support vor Kyosho Minium planes?

### DISCUSSION THREAD

**Discussion by apiening** [pascallanger/DIY-Multiprotocol-TX-Module#498] [General]
Hi There,
I wonder if there is support for the planes of the **Minium** series from the brand **Kyosho**: https://rc.kyosho.com/en/discontinued/dis-rcplane/minium.html

These planes are, unfortunately, discontinued. But they are still around on ebay and I have one in my hangar.

I found **Kyosho** in the protocol details table, but the description says that the protocol has been introduced in 2017. So I don't think it can be the same that was used in the **Minium** planes because they has been introduced around 2010.

Does anyone have more details on that?

With kind regards

Andreas

**Comment by pascallanger** [comment]
Hi Andreas, I would need access to a TX and RX to be able to reverse this protocol. Regards, Pascal

**Comment by apiening** [comment]
Hi Pascal,

these Planes are officially discontinued, but maybe I can find something in good condition on ebay that I could buy that ships to your location.
I'm located in Hamburg, Germany.
Can you PM me your Address for that matter?

With kind regards

Andreas

**Reply by pascallanger** [reply]
Hi Andreas, I don't know where to PM you... Are you on RCGroups? Regards, Pascal

**Reply by pascallanger** [reply]
The protocol is available for test. Please join this issue: https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/issues/658
